cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -215,13 +216,20 @@ def display(self, workspace, figure):
     def measure_objects(self, operand, workspace):
         '''Performs the measurements on the requested objects'''
         objects = workspace.get_objects(operand.operand_objects.value)
+        dimensions = len(objects.shape)
         if objects.has_parent_image:
```

Comment:
I’ve looked at this now and I think I understand.

We test whether there’s a parent image because the parent image may have a mask that needs to be applied before measuring. This is a wonky interface.